src/Bicep.Cli/Commands/TestCommand.cs

```diff
@@ -75,5 +74,32 @@ public async Task<int> RunAsync(TestArguments args)
         }
 
         private bool IsBicepFile(string inputPath) => PathHelper.HasBicepExtension(PathHelper.FilePathToFileUrl(inputPath));
+        private void LogResults(Validation validation){
+            foreach(var (name, evaluation) in validation.SuccessfullEvaluations){
+                logger.LogInformation($"[✓] Evaluation {name} Passed!");
```

Comment:
We should normalize casing for the validation messages.
```suggestion
                logger.LogInformation($"[✓] Evaluation {name} passed!");
```